charts/merlin/values-e2e.yaml

```diff
@@ -50,10 +51,23 @@ merlin:
         min_replica: 0
         max_replica: 1
         cpu_request: "25m"
-        memory_request: "128Mi"
+        memory_request: "64Mi"
 
   imageBuilder:
-    clusterName: "dev"
+    clusterName: merlin-cluster
+    namespace: "mlp"
+    timeout: "10m"
+    kanikoImage: "gcr.io/kaniko-project/executor:v1.6.0"
+    baseImage: ""
+    dockerfilePath: "./Dockerfile"
+    predictionJobDockerfilePath: "docker/app.Dockerfile"
+    contextSubPath: "python/pyfunc-server"
+    predictionJobContextSubPath: "python/batch-predictor"
+    buildContextURI: "git://github.com/gojek/merlin.git#refs/tags/v0.1"
+    predictionJobBuildContextURI: "git://github.com/gojek/merlin.git#refs/tags/v0.1"
```

Comment:
Why do we use v0.1 tag? Should we use newer tag here?

Reply:
That value is just placeholder, it will be overriden by deploy-merlin.sh